gulp/tasks/clean.js

```diff
@@ -1,21 +1,21 @@
+/* eslint-env node */
```

Comment:
I thought you had a rule so that the gulp directory was automatically treated as being a node environment?